qa/L0_warmup/test.sh

```diff
@@ -65,7 +65,7 @@ for BACKEND in ${BACKENDS}; do
     cp -r /data/inferenceserver/${REPO_VERSION}/qa_model_repository/${BACKEND}_float32_float32_float32 models/. && \
     cp -r /data/inferenceserver/${REPO_VERSION}/qa_sequence_model_repository/${BACKEND}_sequence_int32 models/.
 
-    INPUT_PREFIX="INPUT" && [ "$BACKEND" == "libtorch" ] && INPUT_PREFIX="INPUT__"
+    INPUT_PREFIX="INPUT"
```

Comment:
Since INPUT_PREFIX is being used for both `*_zero_1_object` [identity] models and `*_float32_float32_float32` [addsub] models you should create a new variable for IDENTITY_INPUT_PREFIX as
```
IDENTITY_INPUT_PREFIX ="INPUT" && [ "$BACKEND" == "libtorch" ] && IDENTITY_INPUT_PREFIX="INPUT__"
```
to handle this case.

This way the INPUT_PREFIX change is only for addsub models and stands correct.

Reply:
Thanks for clarifying, working on it now.